gitbook-docs/market/cmd-reference/ask.md

```diff
@@ -61,27 +66,28 @@ Output
 }
 ```
 
-## Retrieve Ask
+## Retrieving info on proof request
+### Usage
 
-```
-python3 ask_tools.py get --h
-usage: ask_tools.py get [-h] [--key KEY]
+Proof requesters can check their requests with the following:
+ 
+```console
+foo@bar:~$ python3 request_tools.py get -h
```

Comment:
The `foo@bar` part actually doesn't provide information. Yep, there will be some users on some machines. That's always so.

It's enough to have `$` or `#` in the beginning of a line to have the `console` formatting.

Reply:
But I like that you've joined each example's input and output parts. It just looks better this way.

Reply:
Fixed